src/main/java/org/mifos/connector/mpesa/camel/routes/PaybillRoute.java

```diff
@@ -199,6 +200,7 @@ public void configure() {
                     variables.put("phoneNumber", paybillConfirmationRequestDTO.getMsisdn());
                     variables.put("mpesaTransactionId", mpesaTransactionId);
                     variables.put(TRANSACTION_ID, transactionId);
+                    variables.put(TRANSFER_CREATE_FAILED, false);
```

Comment:
how does it move to completed
?

Reply:
This is after firing the confirmation request.
Validation : IN_PROGRESS
Confirmation : COMPLETED